.github/workflows/production.yml

```
@@ -0,0 +1,75 @@
+      run: |
+        if [[ ! "${{ github.event.inputs.commit_tag }}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+          echo "Invalid commit tag format. Expected format: <number>.<number>.<number>"
+          exit 1
+        fi
+
+    - uses: actions/checkout@v4
+      with:
+        ref: main
+        fetch-depth: 0
+
+    - uses: actions/setup-node@v2
```

actionlint (1.7.4)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)